_source/_posts/2021-12-18-native-java-helidon.adoc

```diff
@@ -0,0 +1,403 @@
+---
+layout: blog_post
+title: "Build REST APIs and Native Java Apps with Helidon"
+author: matt-raible
+by: advocate
+communities: [java]
+description: "Learn how to create REST APIs and native images with Helidon, a Java microservices framework from Oracle."
+tags: [native, java, graalvm, helidon]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+github: https://github.com/oktadev/native-java-examples
+---
+:page-liquid:
+:toc: macro
+:experimental:
+
+Project Helidon is a set of Java libraries for writing microservices. It's an open source, Apache 2.0-licensed, Oracle-sponsored project and contains support for MicroProfile as well as a reactive, functional API. Helidon's two different style are known as Helidon MP and SE, respectively. The https://helidon.io[helidon.io] website says:
+
+====
+Since Helidon is simply a collection of Java libraries running on a fast Netty core, there is no extra overhead or bloat.
```

Comment:
I used the ==== to indicate a quotation. If you look at the preview, you'll see it surrounds the text with a yellow box. I could use > as well, but I don't like the look of that. I don't think linking to Netty is relevant since it's not linked to by Helidon on their site.